e2e/testcafe-devextreme/tests/scheduler/grouping/overflow.ts

```diff
@@ -0,0 +1,80 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import Scheduler from 'devextreme-testcafe-models/scheduler';
+import { createWidget } from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+
+fixture.disablePageReloads`Scheduler: Grouping overflow`
+  .page(url(__dirname, '../../container.html'));
+
+['week', 'month'].forEach((viewType) => {
+  ['vertical', 'horizontal'].forEach((groupOrientation) => {
+    ['hidden', 'allDay'].forEach((allDayPanelMode) => {
+      [[9, 14, 60], [0, 24, 360]].forEach(([startDayHour, endDayHour, cellDuration]) => {
+        const allParams = `${viewType} ${groupOrientation} ${allDayPanelMode} ${startDayHour} ${endDayHour}`;
+        test(`Long appointments should not overflow group view (${allParams})`, async (t) => {
```

Comment:
Super minor comment.

Let's add an empty line here to make test case start more obvious: